code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -50,14 +61,15 @@
 		var/mob/living/carbon/human/H = target
 		H.adjustStaminaLoss(8)
 
-
 /mob/living/simple_animal/hostile/carp/holocarp
 	icon_state = "holocarp"
 	icon_living = "holocarp"
 	maxbodytemp = INFINITY
 	gold_core_spawnable = CHEM_MOB_SPAWN_INVALID
 	del_on_death = 1
 
+/mob/living/simple_animal/hostile/carp/holocarp/Initialize()

```

Comment:
mapload arg.

Reply:
You are right, it would be better to override carp_randomify. Done.